pkg/broker/constants.go

```diff
@@ -1,18 +1,58 @@
 package broker
 
-import "time"
+import (
+	"time"
+)
 
 // CacheTTL TTL for catalog cache record expiry
 var CacheTTL = 1 * time.Hour
 
 var nonCfnParams = []string{
-	"aws_access_key",
-	"aws_secret_key",
 	"region",
 	"target_role_name",
 	"target_account_id",
 }
 
+var nonCfnParamDefs = map[string]interface{}{
```

Comment:
injecting the nonCfnParams centrally, so no need to specify them in the template.